src/Nest/Document/Multiple/Bulk/BulkRequest.cs

```diff
@@ -134,10 +141,11 @@ public BulkDescriptor IndexMany<T>(IEnumerable<T> @objects, Func<BulkIndexDescri
 		) where T : class
 		  where TPartialDocument : class
 		{
-			foreach (var item in @objects)
-				AddOperation(bulkUpdateSelector.InvokeOrDefault(new BulkUpdateDescriptor<T, TPartialDocument>().IdFrom(item), item));
-
-			return this;
+			var operations = @objects
```

Comment:
Talked about this and we'll rewrite to `foreach` loop or consume Enumerator directly